### PR TITLE
KAA-1124 Documentation structure and stubs (gh-pages)

### PR DESCRIPTION
**Pull request by balldir**

### CODE REVIEW

**Comment by kaa-leeroy** [comment]
Can one of the admins verify this patch?

File: doc/Customization-guide/Customizable-system-components/Networking-stack/index.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
 ---
 layout: page
-title: Built-in networking stack
+title: Networking stack
```
Comment (akokhanovskyi) [inline doc/Customization-guide/Customizable-system-components/Networking-stack/index.md]: Let's please rename to "Communication stack", it's more appropriate.

Comment (akokhanovskyi) [inline doc/Customization-guide/Customizable-system-components/Networking-stack/index.md]: Rename file / path as well

Comment (balldir) [inline doc/Customization-guide/Customizable-system-components/Networking-stack/index.md]: @akokhanovskyi Done. I've also updated "Banana Beach documentation structure"

**Comment by forGGe** [comment]
Looks good. Note: documentation contribute guide provided in #724

**Pull request merged**